### ISSUE TITLE
FEATURE REQUEST: Remove the "Lock Hotbar" button

### ISSUE THREAD

**Issue by GambetTV** [saif-ellafi/foundryvtt-minimal-ui#137]
Dunno if this is possible or not, but at some point in v11 they added a new, kind of pointless button to the Hotbar, the "Lock Hotbar" button, which basically is a button that takes up nearly as much room as an actual macro button, but all it does is prevent you from moving around the macro buttons. It'd be great if Minimal UI could add an option to remove this button from view. I figure this makes much more sense as a thing you set and then never touch again, so having it permanently take up space just feels pointless to me.

![image](https://github.com/saif-ellafi/foundryvtt-minimal-ui/assets/72000497/eff8f2f4-b5c4-48d3-93bb-9756b07bda14)

**Comment by saif-ellafi** [comment]
Yeah - it makes sense, thanks for the idea. First I am looking into stopping to break the players list and navigations panel... Foundry is driving me mad with that
